package.json

```diff
@@ -39,15 +42,19 @@
     "make-runnable": "^1.3.6",
     "method-override": "^2.3.10",
     "methods": "^1.1.2",
+    "moment": "^2.24.0",
     "morgan": "^1.9.1",
     "nodemailer": "^6.3.0",
     "passport": "^0.4.0",
     "passport-local": "^1.0.0",
     "pg": "^7.12.1",
+    "pg-hstore": "^2.3.3",
     "regenerator-runtime": "^0.13.3",
     "request": "^2.87.0",
-    "sequelize": "^5.15.0",
+    "sequelize": "^5.15.2",
     "sequelize-cli": "^5.5.0",
+    "sinon": "^7.4.1",
```

Comment:
Why do we have `sinon` and `sinon-chai` in both `dependencies` and `dev-dependencies`
? Please remove it from `dependencies`.

Reply:
It's removed.